File: c/fibonacci_Series.c
#include <stdio.h>
int main()
{

    int n1 = 0, n2 = 1, n3, num;
    printf("Enter the number : ");
    scanf("%d", &num);

    printf("%d\t%d\t", n1, n2);
    for (int i = 0; i < num - 2; i++)
    {
        n3 = n1 + n2;
        n1 = n2;
        n2 = n3;
        printf("%d\t", n3);
    }
}

File: Core_Java/Core_Java/src/day_01/fibonacci_series.java
package day_01;

import java.util.Scanner;

public class fibonacci_series {
	
	public static void printFibonacciSeries(int n) {
		int n1 = 0;
		int n2 = 1;
		System.out.print(n1 + " " + n2 + " ");
		for(int i = 2 ; i < n ; i++) {
			int n3 = n1 + n2;
			System.out.print(n3 + " ");
			n1 = n2;
			n2 = n3;
		}
	}

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter a number : ");
		int n = sc.nextInt();
		printFibonacciSeries(n);
		sc.close();
	}

}
